python/drned_xmnr/op/base_op.py

```diff
@@ -174,16 +185,16 @@ def run_with_trans(self, callback, write=False):
     def extend_timeout(self, timeout_extension):
         dp.action_set_timeout(self.uinfo, timeout_extension)
 
-    def proc_run(self, proc, outputfun, timeout):
-        fd = proc.stdout.fileno()
+    def proc_run(self, outputfun, timeout):
+        fd = self.drned_process.stdout.fileno()
         fl = fcntl.fcntl(fd, fcntl.F_GETFL)
         fcntl.fcntl(fd, fcntl.F_SETFL, fl | os.O_NONBLOCK)
 
         stdoutdata = ""
-        while proc.poll() is None:
+        while self.drned_process.poll() is None:
             rlist, wlist, xlist = select.select([fd], [], [fd], timeout)
             if rlist:
-                buf = proc.stdout.read()
+                buf = self.drned_process.stdout.read()
                 if buf is not None and len(buf) != 0:
                     data = text_data(buf)
                     self.log.debug("run_outputfun, output len=" + str(len(data)))
```

Comment:
I find  `self.log.debug("run_outputfun, output len=%s",str(len(data)))` better as `str(len(data))` is not evaluated when debug is turned off.